static/src/stylesheets/print.scss

```diff
@@ -42,15 +42,22 @@ figure {
 }
 
 * {
-    width: 100% !important;
     max-width: 100% !important;
     margin: 0 !important;
     padding: 0 !important;
     border: 0 !important;
     min-height: 0 !important;
-    height: auto !important;
     background-color: #ffffff !important;
     color: #000000 !important;
+
```

Comment:
Would we perhaps want to exclude these dimension coercions for SVG children generally?